GUIs/ScriptRunnerPyQt5_GUI/scripFunc/scripWINONLY.py

```diff
@@ -524,13 +549,195 @@ def calladdusrtosys():
+                    EXEC = sub.Popen(["powershell", "& {net localgroup users}"], stdout = sub.PIPE)
+                    stdout, _ = EXEC.communicate()
+                    output = stdout.decode("utf-8")
+                    output = output.split("\n")
+                    # print(output)
+                    n = []
+                    for i in range(6, len(output) - 3):
+                        # print(output[i])
+                        # if '\r' in output[i]:
+                        #     print('found in ' + str(i))
+                        #     print(output[i])
+                        n.append(output[i])
+                    a = []
+                    for i in range(0, len(n)):
+                        x = n[i]
+                        y = x.split('\r')
+                        # print(y)
+                        a.append(y)
+                    # print(a)
+                    names = []
+                    for i in range(0, len(a)):
+                        # print(a[i])
+                        x = a[i]
+                        y = x[0]
+                        # print(y)
+                        names.append(y)
+                    # print(names)
+
+                    # Local Administrators are added to list of names
+                    EXEC = sub.Popen(["powershell", "& {net localgroup administrators}"],
+                                     stdout = sub.PIPE)
+                    stdout, _ = EXEC.communicate()
+                    output = stdout.decode("utf-8")
+                    output = output.split("\n")
+                    # print(output)
+                    n2 = []
+                    for i in range(6, len(output) - 3):
+                        # print(output[i])
+                        # if '\r' in output[i]:
+                        #     print('found in ' + str(i))
+                        #     print(output[i])
+                        n2.append(output[i])
+                    a2 = []
+                    for i in range(0, len(n2)):
+                        x = n2[i]
+                        y = x.split('\r')
+                        # print(y)
+                        a2.append(y)
+                    # print(a)
+                    for i in range(0, len(a2)):
+                        # print(a[i])
+                        x = a2[i]
+                        y = x[0]
+                        EXEC = sub.Popen(["powershell", "& {$env:UserName}"], stdout = sub.PIPE)
+                        stdout, _ = EXEC.communicate()
+                        output = stdout.decode("utf-8")
+                        output = output.split('\r\n')
+                        if output[0] == y:
+                            y = output[0] + '   (Current User)'
+                        # print(y)
+                        names.append(y)
+                    # print(names)
+                    return names
+
+                listoNames = findnames()
+
+                for i in range(0, len(listoNames)):
```

Comment:
Codacy found an issue: [Consider using enumerate instead of iterating with range and len](https://app.codacy.com/manual/vipersniper0501/CP_Scripts2/pullRequest?prid=5927260)